board/miyoo/boot/ipk_install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+export TERM=linux
+WARN="
+MiyooCFW team is not responsible for any issues \
+arising from the use of third-party applications \
+with current software. Support, bug fixes, or \
+improvements for 3rd-party apps are not provided \
+by our team members unless explicitly stated otherwise \
+(contact Maintainer of the app).
+\n\n
+\Z5 Do you still wish to CONTINUE with install?\Zn
+\n\n
+   Select an option & press START"
+ipk_directory="/mnt/pkgs"
+package_list=()
+help_messages=()
+index=1
+for ipk_file in "$ipk_directory"/*.ipk; do
+    if [[ -f "$ipk_file" ]]; then
+	ipk_name=$(basename -s .ipk "$ipk_file")
+	package_list+=("$index" "$ipk_name" "off")
+	help_messages[$((2 * index))]="$index"
+	help_messages[$((2 * index + 1))]="$ipk_file"
+	((index++))
+    fi
+done
+if [[ ${#package_list[@]} -eq 0 ]]; then
+    echo "No packages to install."
+    exit 1
+fi
+
+dialog --backtitle "MiyooCFW 2.0" --timeout 10 --title "IPK package manager" --colors --no-shadow --msgbox "\nSelect pkg with \Zb\Z3Y\Zn button on the list." 8 28
+until selected_packages=$(dialog --backtitle "MiyooCFW 2.0" --title "IPK package manager" --colors --no-shadow --help-button --help-status --checklist "Press \Zb\Z3Y\Zn to select packages to install:" 20 60 15 "${package_list[@]}" 3>&1 1>&2 2>&3); do
+    case $? in
+	(1) btn=Cancel && break;;
+	(2)
+	    set -- $selected_packages
+	    shift
+	    index=$1
+	    package_info=$(opkg info "${help_messages[$((2 * index + 1))]}")
+	    package_name=$(echo "$package_info" | grep "^Package:" | cut -d' ' -f2)
+	    package_description=$(echo "$package_info" | grep "^Description:" | cut -d' ' -f2-)
```

Comment:
Last thing: 
Version, Source, Maintainer to HELP info at least.

and should be good for now.